src/resources/tenants/index.ts

```diff
@@ -23,7 +23,7 @@ export class Tenants {
 
   async set<T = CommonMetadata>(
     id: string,
-    tenantData: SetTenant,
```

Comment:
Just renamed this to be consistent with the name of the `SetObjectProperties` interface.